components/patterns/navs/nav-mobile.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <nav
+    class="bg-mtl-black-500 fixed top-0 left-0 w-full h-12 flex justify-between items-center px-2"
+  >
+    <IconMenuOutline
+      @click.native="$emit('openNav')"
+      size="1x"
+      class="text-mtl-white w-6 h-6"
+    />
+    <IconNotifications v-if="true" size="1x" class="text-mtl-white w-6 h-6" />
+    <IconNotificationsActive v-else size="1x" class="text-mtl-white w-6 h-6" />
+  </nav>
+</template>
+
+<script>
+import {
+  IconMenuOutline,
+  IconNotificationsActive,
+  IconNotifications
```

Comment:
do we import order? it should be 
```
  IconMenuOutline,
  IconNotifications,
  IconNotificationsActive,
```

Reply:
We already have a rule for import i am not sure about what you mean ?